### ISSUE TITLE
report httpd tracebacks to sentry

### ISSUE THREAD

**Issue by TomasTomecek** [packit/packit-service#1697]
Triggered by https://github.com/packit/packit-service/issues/1696

Since the issue wasn't submitted to sentry and
1) Jan had to ping me
2) and I had to find the traceback in logs (like a caveman)

we should also feed sentry with tracebacks that occur in httpd in our API -  we have no clue how many 500s happen in our side AFAIK

**Comment by mfocko** [comment]
Is this still relevant after Jirka's PR for splunk logs gets merged?

**Comment by TomasTomecek** [comment]
absolutely: we don't have any alerts on logs right now so we wouldn't know that something is crashing our httpd

**Comment by majamassarini** [comment]
For an example of exception raised by flask code (which I suppose is ran by httpd) and not logged by sentry see description of packit/packit-service#1958

**Comment by jpopelka** [comment]
OTOH it seems to work in [this case](https://red-hat-it.sentry.io/issues/4065286124)
